src/components/BlogPost.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useEffect } from "react";
+import Markdown from 'markdown-to-jsx';
+
+const fm = require('front-matter');
+
+const articleFake = {
+  tags: ['nobodycares', 'irrelevant', 'so?', 'sowhat', 'whatever'],
+  likes: '420'
+};
+
+function MdPost() {
```

Comment:
Minor, and to be fair this should be configured as a lint rule, but for consistency, can we use arrow functions? Also can we clarify the name a bit?

So, suggesting:

`const MarkdownPost = () => {`

and of course change the Export name